src/components/Camera/style.css

```diff
@@ -1,3 +1,36 @@
+@import (less) "../Theme/constants.css";
+
+.rectangle {
+  //default value of react-webcam https://github.com/cezary/react-webcam
+  @width-frame: 640;
+  @height-frame: 480;
+
+  //Spec of overlay
+  @width-percent: 80%;
+  @aspect-ratio: 1.57;
```

Comment:
why the move? isn't there style differences between document camera and face camera?